commands/media.py

```diff
@@ -77,6 +75,15 @@ def watchlist_delete_media(media_name, watchlist_name):
 
 #Choose a media to watch at random from a watchlist
 def watchlist_choose(watchlist_name):
```

Comment:
I personally think choose, clear, view watchlist should be in watchlist file cause they don't really need media info. But idk its okay if you think not!

Reply:
[Discussed in chat] - Decided to keep it as is since media information is being accessed.